build/build.js

```diff
@@ -1,22 +1,26 @@
-// https://github.com/shelljs/shelljs
 'use strict'
 
-require('shelljs/global')
-env.NODE_ENV = 'production'
+require('./check-versions')()
+
+process.env.NODE_ENV = 'production'
 
-const path = require('path')
-const config = require('../config')
 const ora = require('ora')
+const path = require('path')
+const chalk = require('chalk')
+const shell = require('shelljs')
 const webpack = require('webpack')
+const config = require('../config')
 const webpackConfig = require('./webpack.prod.conf')
 
 const spinner = ora('building for production...')
 spinner.start()
 
 const assetsPath = path.join(config.build.assetsRoot, config.build.assetsSubDirectory)
-rm('-rf', assetsPath)
-mkdir('-p', assetsPath)
-cp('-R', 'assets/', assetsPath)
+shell.rm('-rf', assetsPath)
+shell.mkdir('-p', assetsPath)
+shell.config.silent = true
+shell.cp('-R', 'assets/*', assetsPath)
+shell.config.silent = false
```

Comment:
these two lines (line 21 and 23) will stops progress increasing at `17%`. Removed and rebased.